raiden/tests/utils/eth_node.py

```diff
@@ -523,14 +510,13 @@ def run_private_blockchain(
         )
 
         if node.miner:
-            config['unlock'] = 0
+            config['unlock'] = to_checksum_address(config['address'])
```

Comment:
Why do you want to unlock the miner account?

Reply:
No idea why that's done. I just converted it to use the explicit address instead of the index in the keystore since geth was giving a deprecation warning that unlock by index is doing away.

Reply:
Hummm, maybe we should remove it, otherwise `eth_sendTransaction` will work and that is sort of unexpected.

Reply:
I'll remove it and see if anything breaks

Reply:
Doesn't work without `unlock`. Apparently geth requires the Clique miner account to be unlocked:
```
WARN [04-15|17:22:01.283] Block sealing failed                     err="authentication needed: password or unlock"
```